lib/eventemitter2.js

```diff
@@ -182,32 +194,20 @@
         if (!tree._listeners) {
           tree._listeners = listener;
         }
-        else if(typeof tree._listeners === 'function') {
-          tree._listeners = [tree._listeners, listener];
-        }
-        else if (isArray(tree._listeners)) {
+        else {
+          if (typeof tree._listeners === 'function') {
+            tree._listeners = [tree._listeners];
```

Comment:
i didn't, if type is function then i transform it to array and after that make push
it is needed for check memory leak when maxListeners is 1 and we add second listener